flink-yarn/src/main/java/org/apache/flink/yarn/configuration/YarnConfigOptions.java

```diff
@@ -225,6 +226,22 @@
 				.noDefaultValue()
 				.withDescription("Specify YARN node label for the YARN application.");
 
+	public static final ConfigOption<Boolean> REQUIRE_LOCALIZE_KEYTAB =
+		key("yarn.security.kerberos.require.localize.keytab")
```

Comment:
I think this should be called `yarn.security.kerberos.ship-client-keytab` or `yarn.security.kerberos.ship-local-keytab`